src/pages/Character.tsx

```diff
@@ -124,11 +133,27 @@ const Character = () => {
             Witaj, {currentUser?.email}!
```

Comment:
A co jeśli nie będzie emaila? Mamy tutaj "?" więc to możliwe, musimy coś alternatywnego ustawić na ten przypadek.